docs/performance_tuning.md

```diff
@@ -150,6 +150,7 @@ The default value is 1 second which ensures prompt response to creating new mode
   or even disable it. For example, with cloud storage, it could cause a cost for API calls to the storage cloud provider. Detecting new versions 
   can be disabled with a value `0`.
 
+- While collecting metrics has negligible performance overhead when used with typical models, when used with lightweight, fast models, the metric incrementation can consume noticeable proportion of CPU time compared to actual inference. Take it into account while enabled metrics for such models. It applies also for the DEBUG log level which could increase the load for a very high throughout use cases.
```

Comment:
Here we could also be more precise about "typical models". Maybe we could also mention `metrics_enable` parameter, so each bullet point in this section addresses the use specific parameter? 
I would also split `log_level` and put it as a separate point so it's more visible.

Reply:
fixed